apollo-router/src/plugins/telemetry/config.rs

```diff
@@ -55,8 +55,8 @@ impl<T> GenericWith<T> for T where Self: Sized {}
 #[serde(deny_unknown_fields, rename_all = "snake_case")]
 pub struct Conf {
```

Comment:
Not this change, but It is irritating that we have to make things like this public for just because they are used in integration tests...